Vagrantfile

```diff
@@ -0,0 +1,32 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = '2'
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box = 'ubuntu/trusty64'
+
+  # Allow the project directory to be accessible inside the Vagrant box.
+  # This should match the Ansible host_vars/vagrant synced_folder value.
+  config.vm.synced_folder '.', '/mnt/vagrant'
+
+  # Ideally, this IP will be unique, so the entry added to /etc/hosts won't
+  # conflict with that of another project.
+  config.vm.network :private_network, ip: '192.168.33.101'
+
+  # Automatically add an entry to /etc/hosts for this Vagrant box (requires
+  # sudo). This should match the Ansible inventory/vagrant ansible_ssh_host
+  # value.
+  config.hostsupdater.aliases = ['bots.loc']
+
+  # A specific name looks much better than "default" in ansible output.
+  config.vm.define 'vagrant'
+
```

Comment:
Can we add:
```
config.vm.provider "virtualbox" do |v|
  v.memory = 1024
  v.cpus = 2
end
```

...the default is 1 cpu w/ 512mb and bumping this will make things a bit faster. I think it's safe to assume any dev has at least those amount of resources available for a machine.